easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v1706-cleanup.patch

```diff
@@ -0,0 +1,406 @@
+--- OpenFOAM-v1706.orig/applications/utilities/mesh/manipulation/setSet/Allwmake	2017-06-30 10:50:14.000000000 +0200
```

Comment:
Where do the explanation go? I've seen some patches with `#` comments, but this seems to be fairly non-standard.